docs/build-for-developers/cli-usage.md

```diff
@@ -4,10 +4,15 @@ sidebar_label: Basic usage
 slug: /cli-usage
 ---
 
-This page shows common usage examples for the CLI.
+This page shows examples for some of the most common usages of the CLI, such as:
```

Comment:
Yes, this is an improvement for sure. Can we maybe say `including:` instead of `such as:`? 

To be fair this stuff is also in the side nav menu, so I wouldn't be against removing this whole intro paragraph :shrug:  It's not adding much is it?